internal/manifest/manifest.go

```diff
@@ -25,72 +23,6 @@ type Manifest struct {
 	SeedshareOwnerPubKeys   []HexString
 }
 
-// ReferenceValues contains the workload independent reference values.
-type ReferenceValues struct {
-	SNP SNPReferenceValues
-	// TrustedMeasurement is the hash of the trusted launch digest.
-	TrustedMeasurement HexString
-}
-
-// SNPReferenceValues contains reference values for the SNP report.
-type SNPReferenceValues struct {
-	MinimumTCB SNPTCB
-}
-
-// SNPTCB represents a set of SNP TCB values.
-type SNPTCB struct {
-	BootloaderVersion *SVN
-	TEEVersion        *SVN
-	SNPVersion        *SVN
-	MicrocodeVersion  *SVN
-}
-
-// SVN is a SNP secure version number.
-type SVN uint8
-
-// UInt8 returns the uint8 value of the SVN.
-func (s *SVN) UInt8() uint8 {
-	return uint8(*s)
-}
-
-// MarshalJSON marshals the SVN to JSON.
-func (s SVN) MarshalJSON() ([]byte, error) {
-	return []byte(strconv.Itoa(int(s))), nil
-}
-
-// UnmarshalJSON unmarshals the SVN from a JSON.
-func (s *SVN) UnmarshalJSON(data []byte) error {
-	var value float64
-	if err := json.Unmarshal(data, &value); err != nil {
-		return err
-	}
-
-	if value < 0 || value > 255 { // Ensure the value fits into uint8 range
-		return fmt.Errorf("value out of range for uint8")
-	}
-
-	*s = SVN(value)
-	return nil
-}
-
-// HexString is a hex encoded string.
-type HexString string
-
-// NewHexString creates a new HexString from a byte slice.
-func NewHexString(b []byte) HexString {
-	return HexString(hex.EncodeToString(b))
-}
-
-// String returns the string representation of the HexString.
-func (h HexString) String() string {
-	return string(h)
-}
-
-// Bytes returns the byte slice representation of the HexString.
-func (h HexString) Bytes() ([]byte, error) {
-	return hex.DecodeString(string(h))
-}
-
```

Comment:
why does this need to be moved? Moving code around makes it super hard to review and causes loss of history at nearly no benefit (as its destination is still in the same package).

Reply:
To not have a super large file, which makes editing hard

Reply:
I don't see how 60 lines make a file difficult to edit. If you need to do a refactoring, do it as initial work before implementing your feature, in a separate commit. These are exactly the changes that unnecessary blow up the size of your PRs and make the whole thing unpleasant to review.